trunk/SUAVE/Methods/Performance/V_n_diagram.py

```diff
@@ -0,0 +1,477 @@
+
+    return airspeeds_pos, airspeeds_neg
+#------------------------------------------------------------------------------------------------------------
+
+#----------------------------------------
+# Calculate Stall line load factors
+#----------------------------------------
+def stall_line(load_factors, airspeeds, upper_bound, lower_bound, Num_of_points, density_ratio, \
+               rho, lift_coefficient, weight, reference_area):
+    
+    delta = (airspeeds[upper_bound] - airspeeds[lower_bound]) / (Num_of_points + 1)     # Step size
+    for i in range(Num_of_points):           
+        coef = lower_bound + i + 1
+        airspeeds = np.concatenate((airspeeds[:coef], [airspeeds[lower_bound] + (i + 1) * delta], airspeeds[coef:]))
+        Vtas = airspeeds[coef] / density_ratio * Units.knots / Units['ft/s']
+        if load_factors[1] > 0:
+            nl = 0.5 * rho * Vtas**2 * reference_area * lift_coefficient / weight
+        else:
+            nl = -0.5 * rho * Vtas**2 * reference_area * abs(lift_coefficient) / weight
+        load_factors = np.concatenate((load_factors[:coef], [nl], load_factors[coef:]))
+    
+    return load_factors, airspeeds
+#--------------------------------------------------------------------------------------------------------------
+
+#--------------------------------
+# Calculate Gust load factors
+#--------------------------------
+def Gust_loads(airspeeds, load_factors, Kg, CLa, weight, wing_loading, reference_area, rho, density_ratio, \
+               lift_coefficient, Vc, Vd, Uref, Uref_dive, Num_of_points, FARflag):
+
+    gust_load_factors = np.zeros(shape=(4));    
+    gust_load_factors[0] = 1;
+    
+    # Cruise speed Gust loads at Va and Vc
+    gust_load_factors[1] = 1 + Kg * CLa * airspeeds[Num_of_points+2] * Uref / (498 * wing_loading)
+    gust_load_factors[2] = 1 + Kg * CLa * Vc * Uref/(498 * wing_loading)
+   
+    # Intersection between positive cruise gust load and Va
+    if abs(gust_load_factors[1]) > abs(load_factors[Num_of_points+2]):
+        print('Intersection between positive cruise gust load and Va')
+        sea_level_rho = rho / density_ratio**2
+        coefs = [709.486 * sea_level_rho * lift_coefficient, -Kg * Uref * CLa, -498 * wing_loading]
+        V_inters = max(np.roots(coefs))
+        load_factor_inters = 1 + Kg * CLa * V_inters * Uref / (498 * wing_loading)
+
+        airspeeds = np.concatenate((airspeeds[:(Num_of_points + 3)], [V_inters], airspeeds[(Num_of_points + 3):]))
+        load_factors = np.concatenate((load_factors[:(Num_of_points + 3)], [load_factor_inters], load_factors[(Num_of_points + 3):]))
+        
+        # continue stall lines
+        Num_of_points_ext = 5;
+        upper_bound = Num_of_points+3;
+        lower_bound = Num_of_points+2;
+        load_factors, airspeeds = stall_line(load_factors, airspeeds, upper_bound, lower_bound, Num_of_points_ext, \
```

Comment:
you can just have a V-n data structure, or also make a stall_line data structure here too